releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,16 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+## 1.1.0.M6 - December 09, 2019
+
+* Added support for AAD, users can now [Azure TokenCredentials][token_credentials]. Token Credential support allows setting Azure Client ID, Azure Tenant ID, and Azure Client Secret through Environment Variables.
```

Comment:
"Users can now..."? Is there a word missing here? Maybe "Users can now use..." or "Users can now access..."?